source/overloading.tex

```diff
@@ -3421,17 +3421,19 @@
 \end{note}
 
 \pnum
+\indextext{type!integral!promoted}%
+\indextext{type!arithmetic!promoted}%
 In this subclause, the term
-\term{promoted integral type}
+\defn{promoted integral type}
 is used to refer to those integral types which are preserved by
-integral promotion (including e.g.
+integral promotion~(\ref{conv.prom}) (including e.g.
 \tcode{int}
 and
 \tcode{long}
 but excluding e.g.
 \tcode{char}).
 Similarly, the term
-\term{promoted arithmetic type}
+\defn{promoted arithmetic type}
 refers to floating types plus promoted integral types.
```

Comment:
Oh, right, I was thinking of the default argument promotions, not the arithmetic promotions.